src/presenter/filter-presenter.js

```diff
@@ -41,20 +46,101 @@ export default class FilterPresenter{
     this.#filterModel = filterModel;
     this.#tripPointsModel = tripPointsModel;
     this.#sortModel = sortModel;
+    this.#offersModel = offersModel;
+    this.#destinationsModel = destinationsModel;
 
     this.#tripPointsModel.addObserver(this.#setSortFilterDefault);
     this.#tripPointsModel.addObserver(this.#updateFilterSortView);
+    this.#tripPointsModel.addObserver(this.renderInfo);
 
     this.#filterModel.addObserver(this.#updateSortView);
 
   }
 
   init() {
-    render(new InfoView(), this.#tripHeaderContainer, RenderPosition.AFTERBEGIN);
+    this.renderInfo();
     this.#renderFilterView();
     this.#renderSortView();
   }
 
+  #getTotalTripInfo = () => {
+    const points = this.#tripPointsModel.getPoints();
+    const offers = this.#offersModel.getOffers();
+    const destinaions = this.#destinationsModel.getDestinations();
+    const result = {
+      startPlace: {
+        name: 'loading..',
+        date: '0'
+      },
+      middelPlace: {
+        name: 'loading..'
+      },
+      finalPlace: {
+        name: 'loading..',
+        date: '0'
+      },
+      totalPrice:'loading'
+    };
+    let minDate;
+    let maxDate;
+    let totalPrice = 0;
+    let minDateIndex;
+    let maxDateIndex;
+
+
+    if(points.size !== 0 && Object.keys(offers).length !== 0 && Object.keys(destinaions).length !== 0){
+
+      minDate = Date.parse(points.values().next().value.dateFrom);
+      maxDate = Date.parse(points.values().next().value.dateTo);
+
+      points.entries().forEach((point) => {
```

Comment:
```suggestion
      points.entries().forEach(([id, point]) => {
```
`point[0]` и `point[1]` - ну так себе именование.